static/ident_test.go

```diff
@@ -2,7 +2,6 @@ package static_test
 
```

Comment:
All of the tests in this file are only concerned with identities, so you don't need any of the expectations about type names, message names, handlers, etc.

Reply:
Should be addressed in https://github.com/dogmatiq/configkit/pull/138/commits/cc380589dc54a8fdc602b3489963796fb379594d.